ibeam/src/var.py

```diff
@@ -14,6 +14,29 @@
 CHROME_DRIVER_PATH = os.environ.get('IBEAM_CHROME_DRIVER_PATH')
 """Path to the Chrome Driver executable file."""
 
+SQLITE_DB_PATH = os.environ.get('IBEAM_SQLITE_DB_PATH',
```

Comment:
this is the path used inside docker. with docker-compose, it can be mapped to either docker volume or host file. probably I should use a specific credentials folder instead of sharing the outputs dir.